calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -153,7 +153,7 @@ public void Equals()
                     DisplayContent = MathLogic.Multiply(_number1, _number2).ToString();
                     break;
                 case "/":
-                    if (DisplayContent != "0")
+                    if (_number2 != 0.0)
```

Comment:
This actually brought my attention to line 143 `double _number2 = Convert.ToDouble(DisplayContent);`
According to: https://docs.microsoft.com/en-us/dotnet/api/system.convert.todouble?view=net-5.0 it is possible that this also throws an unhandled exception.

![image](https://user-images.githubusercontent.com/58752614/101564334-f4cfbf80-3998-11eb-863c-99921129f786.png)

Reply:
Another issue #55 was opened to fix it. Isn't better to close this PR and fix this later?